### PR TITLE
refactor - dynamic file handling in cloudinary

### PR DESCRIPTION
**Pull request by MTouthang**

### CODE REVIEW

File: backend/controller/contentController.js [hunk 1]
```diff
@@ -206,17 +205,20 @@ const httpGetContent = asyncHandler(async (req, res, next) => {
   result.contents = await Content.find(query)
     .skip(startIndex)
     .limit(LIMIT)
-    // TODO: test fail, reverted back.
     .sort(sorting.latestContent || sorting.likesCount || sorting.trending);
 
+  if (!result.contents) {
```
Comment (shivamvijaywargi) [inline backend/controller/contentController.js:210]: Will the middleware not stop the user before this?
If yes, then this error message is not correct

File: backend/controller/contentController.js [hunk 2]
```diff
@@ -268,22 +270,25 @@ const httpDeleteById = asyncHandler(async (req, res, next) => {
 
   const { thumbnail, trailer, content } = contentData;
 
-  // FIXME: Fix this as discussed (MANG)
-  // perform delete in cloudinary
   if (contentData) {
```
Comment (shivamvijaywargi) [inline backend/controller/contentController.js]: if (contentData)? Why this? contentData looks like an object...

**Review by shivamvijaywargi — COMMENTED** [review]
I have marked few things in the files, please fix them and also put a check for the array fields, they should be checked in the controller when client sends it.

File: backend/controller/contentController.js (hunk 1 above)
Comment (MTouthang) [reply backend/controller/contentController.js:210]: will change the message "content not able to Fetch"

File: backend/controller/contentController.js (hunk 2 above)
Comment (MTouthang) [reply backend/controller/contentController.js]: this condition check is removed

Comment (shivamvijaywargi) [reply backend/controller/contentController.js]: You have removed this, are you checking the thumbnail, trailer, content then?

**Review by shivamvijaywargi — APPROVED** [review]

**Pull request merged**